controllers/performanceprofile_controller.go

```diff
@@ -342,36 +343,41 @@ func (r *PerformanceProfileReconciler) applyComponents(profile *performancev2.Pe
 		return nil, nil
 	}
 
-	components, err := manifestset.GetNewComponents(profile, profileMCP, &r.AssetsDir)
+	profileInfo := &pinfo.PerformanceProfileInfo{
+		PerformanceProfile:       *profile,
+		WorkloadPartitionEnabled: false,
```

Comment:
When the API for the ```WorkloadPartition``` object will be ready, we will have to watch for the object's existence and change this field accordingly.